.maestro/security_tests/3_-_AddressBarSpoof,_appschemes.yaml

```
@@ -1,4 +1,4 @@
-appId: com.duckduckgo.mobile.android
+appId: com.duckduckgo.mobile.android.debug
```

```suggestion
appId: com.duckduckgo.mobile.android
```

That's only required locally